conda/cli/install.py

```diff
@@ -6,6 +6,10 @@
 from logging import getLogger
 import os
 from os.path import abspath, basename, exists, isdir, isfile, join
+try:
+    from os import scandir
+except ImportError:
+    from scandir import scandir
 
```

Comment:
Before we merge this PR, we should consider moving all of these imports to `conda.common.compat`.

Reply:
Now that CI seems healthier again do you want to go ahead and do this?